app/controllers/induction_programme_choices_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class InductionProgrammeChoicesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :ensure_induction_coordinator
+
+  def show
+    @programme_choices = [
+      OpenStruct.new(id: "funded_training_provider", name: "Use a training provider funded by the Department for Education"),
+      OpenStruct.new(id: "free_development_materials", name: "Use the free development materials"),
```

Comment:
Would it make sense to set IDs as `full_induction_programme` and `core_induction_programme` so we're always using the same terms?

Reply:
Happy to do that but I'm not completely sure which one is which. Would funded_training_provider be cip? What about free_development_materials and the rest?